{{ cookiecutter.repo_name }}/buildSrc/src/main/java/plugins/AppCenterConfigPlugin.kt

```diff
@@ -0,0 +1,49 @@
+package plugins
+
+import com.betomorrow.gradle.appcenter.extensions.AppCenterExtension
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import utils.GitHelper
+import utils.propOrEmpty
+
+class AppCenterConfigPlugin : Plugin<Project> {
+    override fun apply(target: Project) {
+        if (System.getenv("CI") == "true") {
+            target.plugins.apply("com.betomorrow.appcenter")
+            val extension = target.extensions.getByType(AppCenterExtension::class.java)
+            try {
+                extension.configure(target)
+            } catch (e: Exception) {
+                e.printStackTrace()
+            }
+        }
+    }
+
+    private fun AppCenterExtension.configure(target: Project) {
+        val gitHelper = GitHelper(target.rootDir)
+
+        apiToken = target.propOrEmpty("APPCENTER_TOKEN")
+        ownerName = "RightpointApps"
+        notifyTesters = false
+        apps {
+            register("developRelease") {
```

Comment:
I was wondering why you are using register here, instead of just ```developRelease{ }``` and letting the "com.betomorrow.appcenter" plugin creating the tasks.

Reply:
I'm using Kotlin here and I believe the `developRelease { ... }` syntax is Groovy-specific. Because of that I can't take advantage of a lot of non-typed, magic that's happening behind the scenes with Groovy.

Reply:
make sense.